fastlane/spec/fixtures/fastfiles/tool_name_formatting.txt

```diff
@@ -0,0 +1,27 @@
+Some lines in this file should start with a special formatting (the same used in CONTRIBUTING.md), so they are ignored by the name formatting checks.
+
+- ❌ "Use Fastlane to automate your screenshots."
+- ❌ "Use _Fastlane_ to automate your screenshots."
+- ❌ "Fastlane helps you deliver faster."
+- ❌ "Match makes code signing management easy."
+Use _fastlane_ to automate your screenshots.
+_fastlane_ helps you deliver faster.
+_match_ makes code signing management easy.
+- ❌ "`fastlane` is an all-in-one tool for app automation."
+- ❌ "**fastlane** is an all-in-one tool for app automation."
+- ❌ "fastlane is an all-in-one tool for app automation."
+- ❌ "<ins>fastlane</ins> is an all-in-one tool for app automation."
+- ❌ "`match` makes code signing management easy."
+_fastlane_ is an all-in-one tool for app automation.
+_match_ makes code signing management easy.
+FastLane makes code signing management easy. Unfortunately we can't catch this brand misalignment because there's no delimiter around the keyword.
+Fastlane makes code signing management easy. Unfortunately we can't catch this brand misalignment because there's no delimiter around the keyword.
```

Comment:
Wait is this still true? Maybe we could adjust the Regexp to catch this case of "left delimiter = start of line and right delimiter = space" and vice-versa?

Like instead of using `\1` in the Regexp to force delimiters on both sides to be the same, catch them separately then check the values (ie "either are the same or one is space and other is beginning/end of line") in the body of the `scan` block?

Or maybe more simply, run `scan` not on the `line`, but on `" #{line} "` instead?

Reply:
Fixed in 2079206

Reply:
😲 great solution! 👏